polyply/tests/test_assign_chirality.py

```diff
@@ -0,0 +1,243 @@
+# Copyright 2020 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Test assignment of chirality
+"""
+import pytest
+import numpy as np
+import networkx as nx
+from polyply.src.assign_chirality import (_determine_priority,
+                                         tag_chiral_centers,
+                                         Chirality)
+
+def alanine_all_atom(chirality="R"):
+    """
+    Molecule graph of Alanine
+
+       H    O
+       |   /
+    CB-CA-C
+       |   \
+       NH2  O-H
+    """
+    molecule = nx.Graph()
+    molecule.add_nodes_from(["CB", "CA", "NH2", "H", "C", "O", "OH"])
+    molecule.add_edges_from([("CB", "CA"), ("CA", "H"),  ("CA", "NH2"),
+                             ("CA", "C"), ("C", "O"),  ("C", "OH")])
+    nx.set_node_attributes(molecule, {"CB":12, "CA":12, "NH2":17, "H":1, "C":12,
+                                      "O":16, "OH":17}, "mass")
+    nx.set_node_attributes(molecule, {"CB":"C", "CA":"C", "NH2":"N", "H":"H", "C":"C",
+                                      "O":"O", "OH":"O"}, "atomname")
+
+    nx.set_edge_attributes(molecule, {("CB", "CA"):1, ("CA", "H"):1,  ("CA", "NH2"):1,
+                                      ("CA", "C"):1, ("C", "O"):1,  ("C", "OH"):1}, "bond_order")
```

Comment:
@pckroon As you see in theses tests, to analyze the chirality one would need a bond order analysis. Vermouth doesn't do that yet, does it?

Reply:
Vermouth doesn't really care about bonds, since you can't get those from PDB files anyway. Unless you start doing super clever geometry analysis coupled to some gnarly quadratic integer programming.